src/fc/fc_conf.h

```diff
@@ -37,6 +37,27 @@ typedef struct _virFCDriverConfig virFCDriverConfig;
 
 typedef struct _virFCDriver virFCDriver;
 
+typedef struct _virFCRateLimiterBandwidth virFCRateLimiterBandiwdth;
+typedef struct _virFCRateLimiterOps virFCRateLimiterOps;
+typedef struct _virFCRateLimiter virFCRateLimiter;
+
+struct _virFCRateLimiterOps {
+    int one_time_burst;
+    int refill_time;
+    int size;
+};
+
+struct _virFCRateLimiterBandwidth {
+    int one_time_burst;
+    int refill_time;
+    int size;
+};
+
+struct _virFCRateLimiter {
+    virFCRateLimiterBandiwdth bandwidth;
+    virFCRateLimiterOps ops;
+};
```

Comment:
I see that they are not used anywhere. I suggest we remove the rate limiter support if it's not used yet, or link it somehow to libvirt (but I don't see support for it)

Reply:
As it is not used, it's a good idea to remove them and maybe in a future PR they can be used for both disk and net devices. WDYT?

Reply:
sounds good